src/commands/edge/drivers/package.ts

```diff
@@ -63,15 +65,51 @@ export default class PackageCommand extends EdgeCommand {
 		upload: flags.string({
 			char: 'u',
 			description: 'upload zip file previously built with --build flag',
-			exclusive: ['build'],
+			exclusive: ['build-only'],
+		}),
+		assign: flags.boolean({
+			char: 'a',
+			description: 'ask user for a channel to assign the driver to after upload',
+			exclusive: ['assign-to', 'build-only'],
```

Comment:
If we start getting even more interesting option cases, we can also make a [custom flags](https://oclif.io/docs/flags#custom-flags) that could be easier to use. I don't see any reason to change this now though.

Reply:
This isn't an option case though..it's just a list of other flags that can't be used at the same time as this one (oclif gives errors to that effect when the user tries to use them together).

Reply:
This was probably on the wrong line. I was speaking generally to the existence of the dual flags for each case such as `assign`, `assign-to` where with a custom flag, one could possibly do it with one. I see them as helpful for more complicated cases though.

Reply:
Oh, interesting. I'm not sure how custom flags would help with that though...not sure how it would be even possible to have a flag sometime take and arg and sometimes not without it being hacky and fragile.

Something I hadn't thought of until now though that might this a little cleaner would be to rename `--assign-to` to `--channel` and `--install-to` to `--hub`. (`--channel` would imply `--assign` and `--hub` would imply `--install` so the user wouldn't have to specify extra parameters.)

Reply:
The more I think about it, the more I like it, so I'm gonna make this change if that sounds good to you.